### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by gustaflindqvist** [alexchee/atom-ruby-define-method#3]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11)
Object.activate (/Users/gustaf/.atom/packages/ruby-define-method/lib/ruby-define-method.coffee:5:9)
```

**Comment by alexchee** [comment]
Thanks for the info!

**Comment by alexchee** [comment]
is this still an issue? I don't see it happening on my version.

**Comment by alexchee** [comment]
This version and all deprecations should be fixed in 0.2.1, hopefully.
